lib/brpoplpush/redis_script/config.rb

```diff
@@ -2,35 +2,49 @@
 
 module Brpoplpush
   module RedisScript
-    # ThreadSafe config exists to be able to document the config class without errors
-    ThreadSafeConfig = Concurrent::MutableStruct.new("ThreadSafeConfig",
-                                                     :debug_lua,
-                                                     :logger,
-                                                     :script_directory)
-
     #
-    # Shared class for dealing with gem configuration
+    # Class holding gem configuration
     #
-    # @author Mauro Berlanda <mauro.berlanda@gmail.com>
-    class Config < ThreadSafeConfig
+    # @author Mikael Henriksson <mikael@mhenrixon.com>
+    class Config
       DEBUG_LUA        = false
-      LOGGER           = ::Logger.new(STDOUT)
+      LOGGER           = Logger.new(STDOUT)
       SCRIPT_DIRECTORY = nil
 
-      #
-      # Returns a default configuration
-      #
-      # @example
-      #   RedisScript::Config.default => <concurrent/mutable_struct/thread_safe_config RedisScript::Config {
-      #   logger: #<Logger:0x00007f81e096b0e0 @level=1 ...>,
-      #   debug_lua: false,
-      #   }>
-      #
-      #
-      # @return [RedisScript::Config] a default configuration
-      #
-      def self.default
-        new(DEBUG_LUA, LOGGER, SCRIPT_DIRECTORY)
+      attr_reader :script_directory, :debug_lua, :logger
+
+      def initialize
+        @script_directory = nil
```

Comment:
This could be an array of directories eventually or an hash `{file_name: :file_path}`

Reply:
How would you do that? You mean that we provide a test directory with a hello world script maybe?